### PR TITLE
LPS-100546 selectedOptions is not available in ie11

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ */
+export function openDisplayPageModal({
+	displayPageName,
+	formSubmitURL,
+	mappingTypes,
+	namespace,
+	spritemap,
+	title
+}) {
+	if (container) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js]: Call `cleanup`

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+				title={title}
+			/>
+		</ClayIconSpriteContext.Provider>,
+		container
+	);
+
+	Liferay.once('destroyPortlet', () => cleanup());
+}
+
+function cleanup() {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js]: Maybe `cleanUp`? (with camelcase)

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+				mappingTypes={mappingTypes}
+				namespace={namespace}
+				onClose={cleanup}
+				title={title}
+			/>
+		</ClayIconSpriteContext.Provider>,
+		container
+	);
+
+	Liferay.once('destroyPortlet', () => cleanup());
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js]: Just pass `cleanUp` without arrow function

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/DisplayPageModal.es.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+import {fetch, navigate} from 'frontend-js-web';
+import ClayButton from '@clayui/button';
+import PropTypes from 'prop-types';
+import React, {useState, useCallback} from 'react';
+import DisplayPageModalForm from './DisplayPageModalForm.es';
+
+const DisplayPageModal = props => {
+	const {formSubmitURL, onClose} = props;
+
+	const form = React.createRef();
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/DisplayPageModal.es.js]: `useRef`

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/modal/openDisplayPageModal.es.js]: mimiimi

**Pull request closed without merging**